src/test/java/com/pubnub/contract/objectV2/members/state/ChannelMembersState.kt

```diff
@@ -4,6 +4,7 @@ import com.pubnub.api.models.consumer.objects_api.member.PNMembers
 
 class ChannelMembersState {
     var channelId: String? = null
-    var memberList: List<PNMembers>? = mutableListOf()
-    var member: PNMembers? = null
+    var resultMemberList: List<PNMembers>? = mutableListOf()
+    var memberToBeAdded: PNMembers? = null
```

Comment:
If you make it list it could support situations when we're adding more than one thing in one scenario. Same applies to removed ones. Not sure if that's what we want/need, but it's an option

Reply:
Yes, there is no support for more than one Member. For know I would leave it as it is. We can easily change it in the future when scenarios will be more complex.